tracing-futures/src/executor.rs

```diff
@@ -56,6 +56,7 @@ impl Instrumented<Runtime> {
     ///
     /// This method simply wraps a call to `tokio::runtime::Runtime::spawn`,
     /// instrumenting the spawned future beforehand.
+    #[cfg(feature = "futures-01")]
```

Comment:
Nit: Since the `impl` block has the `feature = "futures-01"` cfg, we don't need to put it on the individual methods in  that impl.

Reply:
This `impl` block is only guarding on the `tokio` feature.

Reply:
Oh, whoops, I misread the diff. This seems right then!